loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -594,6 +596,70 @@ module BABYLON.GLTF2 {
             var material = getMaterial(runtime, primitive.material);
             multiMat.subMaterials.push(material);
 
+            // Morph Targets
+            if (primitive.targets !== undefined) {
+                morphTargetManager = new BABYLON.MorphTargetManager();
+
+                for (var index2 = 0; index2 < primitive.targets.length; index2++) {

```

Comment:
is using `index2` and `index3`, etc... a common pattern throughout the Babylon codebase?  For smaller loops, anonymous variable names like `i` / `j` (or `index` / `index2`) seem reasonable, but once we get to larger loops like this, it seems that the code legibility could be improved with a more explicit name, like `morphTargetIndex` or something similar...